contrib/python-sambacc.spec

```diff
@@ -0,0 +1,66 @@
+%global bname sambacc
+# set xversion to define the default version number
+%define xversion 0.1
+# set pversion for a customized python package version string
+%{?!pversion: %define pversion %{xversion}}
+# set rversion for a customized rpm version
+%{?!rversion: %define rversion %{xversion}}
+
+
+Name:           python-%{bname}
+Version:        %{rversion}
+Release:        1%{?dist}
+Summary:        Samba Container Configurator
+
+License:        GPLv3+
+URL:            https://github.com/samba-in-kubernetes/sambacc
+# sambacc is not released yet so we're leaving off the url for now
+# once packaged and released we can update this field
+Source:         %{bname}-%{pversion}.tar.gz
+
+BuildArch:      noarch
+BuildRequires:  python3-devel
+
+%global _description %{expand:
+A library and CLI tools for managing Samba servers in containers.
+}
+
+%description %_description
+
+%package -n python3-%{bname}
+Summary: %{summary}
+# Distro requires that are technially optional for the lib
+Requires: python3-samba
+Requires: python3-pyxattr
+
+%description -n python3-%{bname}  %_description
+
+
+%prep
+%autosetup -p1 -n %{bname}-%{pversion}
+
+%generate_buildrequires
+%pyproject_buildrequires -e py3
+
+
+%build
+%pyproject_wheel
+
+
+%install
+%pyproject_install
+%pyproject_save_files %{bname}
+
+
+%check
+%tox -e py3
+
+
+%files -n python3-%{bname} -f %{pyproject_files}
+%doc README.*
+%{_bindir}/samba-container
+%{_bindir}/samba-dc-container
+%{_datadir}/%{bname}/examples/
+
+
+%changelog
```

Comment:
You mean when we actually release? Or some placeholder in this PR?  If the latter, could you provide an example?

Reply:
An example would be:
```
%changelog
Fri May 20 2022 John Mulligan <jmulligan@redhat.com - 0.1
- Initial spec file
```

Reply:
Sure, I understand the format. My question was if you expect a changelong entry for these "pre-release" builds or you expect an entry to be added as part of the release process?  Since we've discussed that the first real release is going to be v0.2  I can put in a "phony" entry for 0.1... but I think I'd prefer not to. Thoughts?

Reply:
I agree. Let's have an entry for 0.2 release.